### PR TITLE
Animated selected tab indicator

### PR DESCRIPTION
**Pull request by HansMuller**

### CODE REVIEW

File: sky/packages/sky/lib/widgets/tabs.dart [hunk 1]
```diff
@@ -3,8 +3,12 @@
 // found in the LICENSE file.
 
 import 'dart:math' as math;
+//import 'dart:sky' as sky;
```
Comment (abarth) [inline sky/packages/sky/lib/widgets/tabs.dart]: ?

Comment (HansMuller) [inline sky/packages/sky/lib/widgets/tabs.dart]: This is becoming a habit. Sorry about that.

File: sky/packages/sky/lib/widgets/tabs.dart [hunk 2]
```diff
@@ -396,13 +424,65 @@ class TabBar extends Scrollable {
...
+  }
+
+  void _indicatorAnimationUpdated() {
+    setState(() {
+    });
+  }
+
+  AnimatedRect get _indicatorRect => _indicatorAnimation.variable as AnimatedRect;
+
+  void _startIndicatorAnimation(int fromTabIndex, toTabIndex) {
```
Comment (abarth) [inline sky/packages/sky/lib/widgets/tabs.dart]: toTabIndex -> type pls

**Comment by abarth** [comment]
LGTM

**Pull request merged**